scripts/goimport-file.sh

```diff
@@ -7,10 +7,20 @@ if [ $# != 1 ] ; then
   exit 1
 fi
 
-sed -i '' -e '
+# remove empty lines inside import block via sed
+sed_expression='
   /^import/,/)/ {
     /^$/d
   }
-' $1
+'
+
+case "$OSTYPE" in
+    "linux-gnu"*)
+        sed -i -e "$sed_expression" $1
+        ;;
+    "darwin"*)
```

Comment:
This does assume the version of sed available is the default on each platform - many users use gnu sed on darwin however. This should detect gnu vs non gnu:

```suggestion
sed_version="$(gsed --version 2>/dev/null)"
case "${sed_version}" in
    *gnu*)
        sed -i -e "$sed_expression" $1
        ;;
    *)
```

Reply:
Does `gsed` installation alias itself as `sed`?

Reply:
It doesn't by default, but the brew package does give instructions on how to do so, and it's not an uncommon configuration